pallets/loans/src/lib.rs

```diff
@@ -538,6 +523,30 @@ impl<T: Config> Pallet<T> {
         T::PalletId::get().into_account()
     }
 
+    fn accrue_interest() -> DispatchResult {
+        for currency_id in Self::currencies() {
+            let total_cash = Self::get_total_cash(currency_id);
+            let total_borrows = Self::total_borrows(currency_id);
+            let util = Self::calc_utilization_ratio(total_cash, total_borrows, 0)?;
```

Comment:
Can we replace `0` with `Zero::zero()` everywhere?

Reply:
Feasible. Good advice.